semgrep/semgrep/core_runner.py

```diff
@@ -128,25 +129,11 @@ def _extract_core_output(
         self, rule: Rule, patterns: List[Pattern], core_run: subprocess.CompletedProcess
     ) -> Dict[str, Any]:
         semgrep_output = core_run.stdout.decode("utf-8", errors="replace")
-        semgrep_error_output = core_run.stderr.decode("utf-8", errors="replace")
```

Comment:
So where does the stderr output goes when in normal mode?
Where is it displayed if all this code is removed?

Reply:
Oh my bad. Yes should definitely still keep this code path

Reply:
Hmm good call